docs/app/pages/components/components-sections/select/select-custom/select-custom.component.html

```diff
@@ -0,0 +1,134 @@
+                    <div class="col-md-4 col-sm-12">
+                        <ux-checkbox [(value)]="allowNull">allowNull</ux-checkbox>
+                    </div>
+                </div>
+                <div class="row uxd-customize-row">
+                    <div class="col-md-4 col-sm-12">
+                        <ux-checkbox [(value)]="showBusyIndicator">busy</ux-checkbox>
+                    </div>
+                </div>
+                <div class="row uxd-customize-row">
+                    <div class="col-md-2 col-sm-12">
+                        <label>placeholder</label>
+                    </div>
+                    <div class="col-md-2 col-sm-12">
+                        <input class="form-control" [(ngModel)]="placeholder">
+                    </div>
+                </div>
+                <div class="row uxd-customize-row">
+                    <div class="col-md-4 col-sm-12">
+                        <button class="btn button-primary" (click)="selected = filteredOptionList[0]">Select first element</button>
+                    </div>
+                </div>
+            </ux-accordion-panel>
+        </ux-accordion>
+    </div>
+</div>
+
+<hr>
+
+<p>
+    The <code>select-custom</code> component is a drop-down with flexible content.
+    The following can be used to customize and interact with the component:
+</p>
+
+<uxd-api-properties tableTitle="Inputs">
+    <tr uxd-api-property name="buttonContent" type="TemplateRef<void>">
+        Rendering the selected item on the drop-down button.
+    </tr>
+    <tr uxd-api-property name="dropdownContent" type="TemplateRef<void>">
+        Rendering the complete data to select from on the drop-down panel.
+    </tr>
+
+    <tr uxd-api-property name="selected" type="T">
+        If specified, the items provided will be selected.
+        To programmatically modify the selection, assign a new array object to this property.
+    </tr>
+    <tr uxd-api-property name="hideFilter" type="boolean" defaultValue="false">
+        If <code>true</code>, the filter field is not displayed.
+    </tr>
+    <tr uxd-api-property name="maxHeight" type="number">
+        Maximum height of the drop-down.
+    </tr>
+    <tr uxd-api-property name="busy" type="boolean" defaultValue="false">
+        If <code>true</code>, displays a busy indicator.
+    </tr>
+    <tr uxd-api-property name="placeholder" type="string" defaultValue="Type to filter...">
+        Placeholder to display inside the filter field.<br>
+        <b>Note:</b> The default value is not localized!
+    </tr>
+</uxd-api-properties>
+
+<uxd-api-properties tableTitle="Outputs">
+    <tr uxd-api-property name="filterChange" type="string">
+        Fires whenever the filter term changes, sends new filter string as event.
+    </tr>
+    <tr uxd-api-property name="selectedChange" type="T">
+        This event will emit any time the selection changes. The event will contain the new selected items.
```

Comment:
Above description begins "Fires..."; this begins "This event will emit...". I recommend changing both to "Emits when..." for consistency.

Reply:
Done.